<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProduce</class>
 <widget class="QDialog" name="DialogProduce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCommand">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRun">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>83</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>run/copy</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>314</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelAddons">
      <property name="text">
       <string>Addons</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPrefix">
      <property name="text">
       <string>Prefix</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditPrefix"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelBasename">
      <property name="text">
       <string>Basename</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditBasename"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelFilters">
      <property name="text">
       <string>Filters</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelCompression">
      <property name="text">
       <string>Compression</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelTheme">
      <property name="text">
       <string>Theme</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxTheme"/>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBoxClone">
      <property name="text">
       <string>Clone</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBoxCryptedClone">
      <property name="text">
       <string>Crypted Clone</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="checkBoxScript">
      <property name="text">
       <string>Script</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="checkBoxUnsecure">
      <property name="text">
       <string>Unsecure</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="checkBoxUnsecure_2">
      <property name="text">
       <string>Yolk</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxAddons"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxFilters"/>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxCompression"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonGenerate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>generate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogProduce</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogProduce</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
